fdbserver/MasterInterface.h

```diff
@@ -184,26 +189,50 @@ struct GetCommitVersionRequest {
 	}
 };
 
+struct GetTlogPrevCommitVersionReply {
+	constexpr static FileIdentifier file_identifier = 16683183;
+	VersionVector ssVersionVector;
+	GetTlogPrevCommitVersionReply() {}
+	template <class Ar>
+	void serialize(Ar& ar) {
+		serializer(ar, ssVersionVector);
+	}
+};
+
+struct GetTlogPrevCommitVersionRequest {
+	constexpr static FileIdentifier file_identifier = 16683184;
+	std::set<Tag> tagSet;
```

Comment:
This needs to be a set of tlog Ids (UUIDs?), correct?

Reply:
I am anticipating the set will be made up of `locations` (so int, not uuid), per existing logic e.g. `writeTypedMessage` and `TagPartitionedLogSystem::push`.  I will make it `int` for this PR. The next PR ("send to only affected tlogs") can iterate on that if needed.

Reply:
Yes, it could even be a `short int` (65535 is enough for tlogs), as tlogs are indexed by an array.